activesupport/lib/active_support/cache.rb

```diff
@@ -352,9 +352,14 @@ def read_multi(*names)
       #
       # Returns a hash with the data for each of the names. For example:
       #
-      #   cache.write("bim", "bam")
-      #   cache.fetch_multi("bim", "boom") { |key| key * 2 }
-      #   # => { "bam" => "bam", "boom" => "boomboom" }
+      #   cache.write("new_key", "value")
+      #   cache.fetch_multi("new_key", "unknown_key") do |key|
+      #     "value for: #{key}"
+      #   end
+      #   # => {
+      #   #   "new_key" => "value",
```

Comment:
Same goes for using "value" here. Its harder to read when the fallback is "value for..."